rector.php

```diff
@@ -10,7 +10,7 @@
         __DIR__ . '/src',
     ])
     ->withRootFiles()
-    ->withCodeQualityLevel(0)
+    ->withPreparedSets(codeQuality: true)
```

Comment:
Are we at risk of CI breaking unexpectedly if Rector add new rules to the pre-defined "codeQuality" set that we have not yet adopted? We are only pinned to `^2.0`, so every CI build will run with whatever happens to be the latest 2.x release of Rector at the time. I'm not sure their versioning strategy for changes to the config / implementation of rules?

We should perhaps be explicit about the rules - or alternatively require an explicit Rector version in composer.json and have a job (dependabot or a custom one like we have in behat/gherkin for the gherkin dependency) to update to each new Rector version as a standalone PR so that we can deal with any issues that arise without affecting other branches?